main.go

```diff
@@ -20,17 +20,21 @@ import (
 )
 
 var opts struct {
-	Output       string
-	InputDir     string
-	TemplateFile string
-	Versions     []string
+	Output           string
+	InputDir         string
+	TemplateFile     string
+	IssueRegex       string
+	PullRequestRegex string
+	Versions         []string
 }
 
 func init() {
 	pflag.StringVarP(&opts.InputDir, "input", "i", "changelog", "read input files from `dir`")
 	pflag.StringVarP(&opts.Output, "output", "o", "", "write generated changelog to this `file` (default: print to stdout)")
 	pflag.StringVarP(&opts.TemplateFile, "template", "t", filepath.FromSlash("changelog/CHANGELOG.tmpl"), "read template from `file`")
 	pflag.StringSliceVar(&opts.Versions, "version", nil, "only print `version` (separate multiple versions with commas)")
+	pflag.StringVarP(&opts.IssueRegex, "issue-regex", "", `/.*/.*/issues/(\d+)`, "`regex` for parsing SCM issue URLs")
+	pflag.StringVarP(&opts.PullRequestRegex, "pr-regex", "", `/.*/.*/pull/(\d+)`, "`regex` for parsing SCM pull requests URLs")
```

Comment:
Sorry, I made a mistake, "requests URLs" in the description should be just "request URLs" :/

Reply:
no worries, fixed